docker/iceberg-rest-fixture/Dockerfile

```
@@ -37,6 +37,7 @@ ENV CATALOG_CATALOG__IMPL=org.apache.iceberg.jdbc.JdbcCatalog
 ENV CATALOG_URI=jdbc:sqlite:/tmp/iceberg_catalog.db
 ENV CATALOG_JDBC_USER=user
 ENV CATALOG_JDBC_PASSWORD=password
+ENV CATALOG_JDBC_STRICT__MODE=password
```

Looks like a copy-paste error? :)
```suggestion
ENV CATALOG_JDBC_STRICT__MODE= true
```